.github/workflows/deploy.yml

```
@@ -0,0 +1,75 @@
+name: Deploy to EC2
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  # CI: Lint & Test (既存のワークフローを実行)
+  lint:
+    name: Run Lint
+    uses: ./.github/workflows/lint.yml
```

actionlint (1.7.10)

[error] error while parsing reusable workflow "./.github/workflows/lint.yml": "workflow_call" event trigger is not found in "on:" at line:4, column:3

(workflow-call)